src/components/layout/TopBar.module.scss

```diff
@@ -24,8 +24,16 @@
   align-items: center;
   justify-content: space-between;
   flex-direction: row;
-  padding: 1rem 2rem;
+  padding: 0.7rem 1rem;
   border: 1px solid rgb(188, 169, 204);
```

Comment:
Spalvos turėtų būti naudojamos iš globalaus stiliaus kintamųjų (temos) (sccs variables in vite project)